packages/component-base/test/delegate-focus-mixin.test.js

```diff
@@ -2,25 +2,21 @@ import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
 import { DelegateFocusMixin } from '../src/delegate-focus-mixin.js';
-import { InputController } from '../src/input-controller.js';
```

Comment:
Removed usage of controller and replaced with adding an `<input>` element manually.